_studio/mfx_lib/encode_hw/shared/ehw_task_manager.h

```diff
@@ -102,6 +102,7 @@ namespace MfxEncodeHW
         mfxU16                  m_maxParallelSubmits = 0;
         mfxU16                  m_nTasksInExecution  = 0;
         mfxU16                  m_nRecodeTasks       = 0;
+        bool                    m_bSync              = false;
```

Comment:
flag name is misleading - it is used not to force sync but to repeat async cycle w/ same parameters except query stage,
Please rename to something more appropriate.

Reply:
Fixed